torch/csrc/jit/passes/create_functional_graphs.h

```diff
@@ -10,5 +10,7 @@ TORCH_API void CreateFunctionalGraphs(const std::shared_ptr<Graph>& graph);
 
 TORCH_API void InlineFunctionalGraphs(const std::shared_ptr<Graph>& graph);
 
+TORCH_API void RemoveMutation(const std::shared_ptr<Graph>& graph);
```

Comment:
Can we get a comment describing at a high-level the purpose of this pass, and generally what is a candidate for mutation removal?